py3xui/client/client.py

```diff
@@ -65,5 +67,5 @@ class Client(BaseModel):
     flow: str = ""
     limit_ip: int = Field(default=0, alias=ClientFields.LIMIT_IP)  # type: ignore
     sub_id: str = Field(default="", alias=ClientFields.SUB_ID)  # type: ignore
-    tg_id: str = Field(default="", alias=ClientFields.TG_ID)  # type: ignore
+    tg_id: Optional[int] = Field(default="", alias=ClientFields.TG_ID)  # type: ignore
```

Comment:
Default value is incompatible with the typing.
String can not be assigned as default value for Optional[int].

Reply:
@iwatkot wow, really) default=None is ok ?

Reply:
panel 3x-ui return None if empty value in tg_id

Reply:
**1**
`tg_id: Optional[str] = Field(default=None, alias=ClientFields.TG_ID)`

_validation error for UiInbound
settings.clients.4.tgId
  Input should be a valid string [type=string_type, input_value=511122566, input_type=int]
    For further information visit https://errors.pydantic.dev/2.9/v/string_type_


**2**
`tg_id: Optional[int] = Field(default=None, alias=ClientFields.TG_ID)`

_validation error for UiInbound
settings.clients.0.tgId
  Input should be a valid integer, unable to parse string as an integer [type=int_parsing, input_value='', input_type=str]
    For further information visit https://errors.pydantic.dev/2.9/v/int_parsing_

 default panel return str type

**3**
`tg_id: Optional[int | str] = Field(default=None, alias=ClientFields.TG_ID)`

_Worked perfect_

Reply:
or default = "", i think this worked too

Reply:
@iwatkot

Reply:
File "/root/py-scripts/lib/python3.10/site-packages/py3xui/client/client.py",
line 69, in Client
    tg_id: Optional[int | str] = Field(default=None, alias=ClientFields.TG_ID)
NameError: name 'Optional' is not defined

Reply:
![image](https://github.com/user-attachments/assets/cc090368-255c-47bf-bf55-5b3dbb620e8a)

Reply:
@Goosegit11

> NameError: name 'Optional' is not defined

This is weird, I can clearly see the import of the Optional from typing.
But still, we'll change it to modern type hints.